src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+  description: >
+    JavaScriptのパターンを識別し報告するための、プラグイン可能で設定可能なリンターツールです。
+    コードの品質を簡単に維持することができます。
+  title_format: PAGE_TITLE - ESLint - プラグイン可能なJavaScriptリンターツール
+  skip_to_content: 本文へスキップ
+
+#------------------------------------------------------------------------------
+# Navigation
+#------------------------------------------------------------------------------
+
+navigation:
+  - text: チーム
+    link: team
+  - text: ブログ
+    link: blog
+  - text: ドキュメンテーション
+    link: docs
+  - text: ストアー
+    link: store
+    target: _blank
+  - text: Playground
+    link: playground
+    target: _blank
+
+#------------------------------------------------------------------------------
+# Footer
+#------------------------------------------------------------------------------
+
+footer:
+  title: JavaScriptのコードを修正する準備はできましたか？
+  description: npmからインストール。または今日から寄付を開始する。
+  secondary: Secondary
+  social_icons:
+    title: ソーシャルメディア
+    twitter: Twitter
+    chat: Discord
+    mailing_list: Google グループ
+    github: GitHub
+  theme_switcher:
+    title: 配色モード切り替え
+    light: 通常
+    dark: ダーク
+  language_switcher:
+    title: 言語切替え
+    description: 言語を選択すると、その言語のESLintウェブサイトへ移動します
+    change_language: 言語の変更
+    language: 言語
+  copyright: >
+    Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and ESLint 
+    contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS 
+    Foundation</a> has registered trademarks and uses trademarks.  For a list of 
+    trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please 
+    see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> 
+    and <a href="https://trademark-list.openjsf.org">Trademark List</a>.  
+    Trademarks and logos not indicated on the 
+    <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation 
+    trademarks</a> are trademarks&trade; or registered&reg; trademarks of 
+    their respective holders. Use of them does not imply any affiliation with 
+    or endorsement by them.
+  links:
+    open_jsf: The OpenJS Foundation
+    terms: 利用規則
+    privacy: プライバシーポリシー
+    bylaws: OpenJS Foundation Bylaws
+    trademark: 商標ポリシー
+    trademark_list: 商標リスト
+    cookies: クッキーポリシー
+
+#------------------------------------------------------------------------------
+# 404 Page
+#------------------------------------------------------------------------------
+
+404_page:
+  title: 404エラー
+  subtitle: ページが見つかりません
+  description: 申し訳ございませんが、お探しのページは、移動または消除された可能性があります。
+  actions:
+    back_to_home: ホームページへ
+    browse_docs: ドキュメンテーションを読む
+
+#------------------------------------------------------------------------------
+# Branding Page
+#------------------------------------------------------------------------------
+
+branding_page:
+  title: ブランディング
+  description: 正しいロゴの使い方
+  sections:
+    name:
+      title: 名前
+      description: >
+        ESLint を書くときは、E,SとLは大文字を使用して下さい。「ES」はECMAScriptの略称、Lは「int」の頭文字です。
```

>        ESLint must be written with a capital E, S and L when used, as “ES” stands for ECMAScript and “L” being the start of the word “Lint”.

I think "Lint" is correct. IMHO, "EとSとL" is easy to read.

```suggestion
        ESLint を書くときは、EとSとLは大文字を使用して下さい。「ES」はECMAScriptの略称、Lは「Lint」の頭文字です。
```